vllm/v1/worker/tpu_model_runner.py

```diff
@@ -152,6 +154,15 @@ def __init__(
         self.hidden_size = model_config.get_hidden_size()
         self.vocab_size = model_config.get_vocab_size()
 
+        if self.lora_config is not None:
+            # If LoRA is enabled, we need to disable the recompilation check
+            # because there is currently a small recompilation when we set a
+            # LoRA adapter (_tpu_set_lora / _tpu_reset_lora).
+            # TODO: Remove this recompilation
+            os.environ["VLLM_XLA_CHECK_RECOMPILATION"] = "0"
+            self.check_recompilation = False
```

Comment:
We should be in full control of "VLLM_XLA_CHECK_RECOMPILATION", we don't expect the user to use it. 
Hence we can just set this env var explicitly in the test files or CI when dealing with lora.